solidity/contracts/bridge/Bridge.sol

```diff
@@ -19,14 +19,13 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 
 import {BTCUtils} from "@keep-network/bitcoin-spv-sol/contracts/BTCUtils.sol";
 import {BytesLib} from "@keep-network/bitcoin-spv-sol/contracts/BytesLib.sol";
```

Comment:
It seems we can remove these statements because of the refactor:
```
import {BTCUtils} from "@keep-network/bitcoin-spv-sol/contracts/BTCUtils.sol";
import {BytesLib} from "@keep-network/bitcoin-spv-sol/contracts/BytesLib.sol";
...
    using BTCUtils for bytes;
    using BTCUtils for uint256;
    using BytesLib for bytes;
```

Reply:
Oh yes! https://github.com/keep-network/tbtc-v2/pull/211/commits/8fde88ac94e5566ca0e26c06afb5ded54cf1306d